imap_processing/glows/l1/glows_l1a_data.py

```diff
@@ -0,0 +1,411 @@
+import struct
+from dataclasses import dataclass
+from typing import ClassVar
+
+from imap_processing.glows import version
+from imap_processing.glows.l0.glows_l0_data import DirectEventL0, HistogramL0
+from imap_processing.glows.utils.constants import DirectEvent, GlowsConstants, TimeTuple
+
+
+class DataEverySecond:
+    """Mapping of name to byte for dataeverysecond structure in GLOWS L1A.
```

Comment:
can you expand on dataeverysecond?

Reply:
Sorry, I missed this comment @mstrumik. This documentation is outdated, thanks for catch it. I rewrote the class doc (again) to try and further explain data_every_second and how it is used.